DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/README.md

```diff
@@ -114,141 +151,146 @@ Note that the library files (\*.a) must be included after all of the cpp files i
 
 ### On a Linux* System
 
-1. Generate the `Makefile` by running `cmake`.
-
-    ```bash
-    mkdir build
-    cd build
-    ```
-
-    To compile for the default target (the Agilex® 7 device family), run `cmake` using the command:
-    ```
-    cmake ..
-    ```
-
-    > **Note**: You can change the default target by using the command:
-    >  ```
-    >  cmake .. -DFPGA_DEVICE=<FPGA device family or FPGA part number>
-    >  ```
-    >
-    > Alternatively, you can target an explicit FPGA board variant and BSP by using the following command:
-    >  ```
-    >  cmake .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
-    >  ```
-    >
-    > You will only be able to run an executable on the FPGA if you specified a BSP.
-
-2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
-
-   * Compile for emulation (fast compile time, targets emulated FPGA device):
-
-      ```bash
+1. Change to the sample directory.
+2. Build the program for Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake ..
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake .. -DFPGA_DEVICE=<FPGA device family or FPGA part number>
+   >  ```
+   >
+   > Alternatively, you can target an explicit FPGA board variant and BSP by using the following command:
+   >  ```
+   >  cmake .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
+   >  ```
+   >
+   > You will only be able to run an executable on the FPGA if you specified a BSP.
```

Comment:
```suggestion
   >
   > Alternatively, you can target an explicit FPGA board variant and BSP by using the following command:
   >  ```
   >  cmake .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
   >  ```
   >
   > You will only be able to run an executable on the FPGA if you specified a BSP.
```

we've made this design only work with RTL libraries, so remove this

Reply:
Why can't this tutorial run on a BSP?

Reply:
removed these line manually in latest commit